microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -698,11 +1164,14 @@ Use **Disable local fonts** to make it easier to:
 *  Discover differences between local versions installed on your device and a web font.
 
 
-<!-- moved from "Fix memory problems" article -->
-<!-- todo: move instead to Perf tool article? -->
-<!-- Performance tool >  Memory checkbox -->
 <!-- ====================================================================== -->
 ## Visualize memory leaks (Performance tool: Memory checkbox)
+<!-- This h2 is in downstream version of this doc only.
+todo: move this h2 section to a Perf tool article?
```

Comment:
move "Visualize memory leaks (Performance tool: Memory checkbox)" to a different article?  or move it up next to "View memory metrics" section?

In upstream docs, this h2 section is in:
https://developer.chrome.com/docs/devtools/memory-problems#visualize_memory_leaks_with_timeline_recordings
Moved this h2 section from downstream (redesigned) "Fix memory problems" article.
It would be best to match the upstream location ("Fix memory problems" > "Visualize memory leaks with Performance recordings") - requires revisit design of "Fix memory problems" vs. the upstream Memory > Overview page.
This section is about the **Performance** tool >  **Memory** checkbox - which is doc'd in "View memory metrics", above.

Reply:
low pri, PR is overloaded